Week 2/shopee/src/components/SuccessPage.jsx

```diff
@@ -0,0 +1,57 @@
+                </g>
+              </svg>
+              <h4 className="">
+                <small>|</small> Đơn Hàng
+              </h4>
+            </div>
+          </div>
+          <div className="cart__bg">
+            <hr />
+            <div className="cart__content">
+                <h3 >Đặt Thành Công !</h3>
+            </div>
+          </div>
+          <div className="cart__bg">
+            <div className="cart__content">
+              <div className="order__information">
+                <h5>Tên sản phẩm:{" " + order.title} </h5>
```

Comment:
@phamhuutinh0605 : bỏ mấy cái này đi, chổ khác cũng vậy
```suggestion
                <h5>Tên sản phẩm:{order.title} </h5>
```